prosody/Dockerfile

```diff
@@ -38,6 +38,10 @@ RUN \
     && apt-cleanup \
     && rm -rf /tmp/usr /var/cache/apt
 
+RUN \
+    curl -4so /prosody-plugins/mod_turncredentials.lua \
+    https://raw.githubusercontent.com/netaskd/mod_turncredentials/master/mod_turncredentials.lua
```

Comment:
Why do we need your forked version?

Reply:
Original doesn't support prosody 0.11.X
Changes can be found here:  https://github.com/netaskd/mod_turncredentials/commit/76fabc32159580fac34ab627a016bcb9aa3c0697
Also, @damencho said (https://github.com/jitsi/docker-jitsi-meet/issues/22#issuecomment-564626682) he has plan to push PR his version to jitsi-meet repo. I think we can wait and insert "official" plugin when it will done.

Reply:
Got it, thanks!

Reply:
```dockerfile
ADD https://raw.githubusercontent.com/netaskd/mod_turncredentials/master/mod_turncredentials.lua /prosody-plugins/mod_turncredentials.lua
```
Can you not just use this?

Reply:
I'm very close to preparing my PR for adding turn support by default in jitsi-meet and the module will be part of it and will be installed with jitsi-meet once jitsi-meet-prosody package is installed.

Reply:
https://github.com/jitsi/jitsi-meet/pull/4959

Reply:
Yup, here is the official file:

https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/mod_turncredentials.lua

Reply:
Official : 
`ADD https://hg.prosody.im/prosody-modules/raw-file/tip/mod_turncredentials/mod_turncredentials.lua /prosody-plugins/mod_turncredentials.lua`